pkg/controller/restore_controller.go

```diff
@@ -433,7 +471,7 @@ func (c *restoreController) fetchBackup(bucket, name string) (*api.Backup, error
 	return backup, nil
 }
 
-func (c *restoreController) runRestore(restore *api.Restore, bucket string) (restoreWarnings, restoreErrors api.RestoreResult) {
+func (c *restoreController) runRestore(restore *api.Restore, bucket string, backup *api.Backup) (restoreWarnings, restoreErrors api.RestoreResult, restoreFailure error) {
```

Comment:
I don't think the new `backup` param is actually used anywhere

Reply:
I think I see the mismatch - I kept L481 in rebasing. I was trying to keep it around and save a fetch. If the backup's not returned by `completeAndValidate`, we'll be doing a second fetch in this case.

I'm fine with leaving it as is for now and not passing a backup in, though it does seem a little inefficient and a duplication of work.